frontend/src/scenes/cohorts/cohortLogic.ts

```diff
@@ -273,7 +273,7 @@ export const cohortLogic = kea<cohortLogicType<CohortLogicProps>>([
 
     listeners(({ actions, values, key }) => ({
         deleteCohort: () => {
-            cohortsModel.findMounted()?.actions.deleteCohort(values.cohort)
+            cohortsModel.findMounted()?.actions.deleteCohort({ ...values.cohort, filters: undefined })
```

Comment:
Not sure if I'm using the right keys here, but I think being explicit here is less confusing :) 

```suggestion
            cohortsModel.findMounted()?.actions.deleteCohort({ id: values.cohort.id, name: values.cohort.name })
```